config/fastly/recv.vcl

```diff
@@ -41,12 +41,6 @@ if (req.http.x-forwarded-for) {
     set req.http.X-Forwarded-For = client.ip;
 }
 
-# Normally, you should consider requests other than GET and HEAD to be uncacheable
-# (to this we add the special FASTLYPURGE method)
-if (req.method != "HEAD" && req.method != "GET" && req.method != "FASTLYPURGE") {
-  return(pass);
-}
-
```

Comment:
Any reason for this change 🤔 

We want never cache anything else than HEAD|GET

Reply:
See https://github.com/shopware/docs/pull/547#discussion_r851091947

Reply:
This is part of the standard Fastly template that will be included when you generate your VCL. There is no need to additionally add it as it is the default behavior.

The drawback of doing it right there is that it may be subvert backend selection that you may later make since it's an early return.